dask/dataframe/core.py

```diff
@@ -2464,24 +2464,46 @@ def describe(
         percentiles_method="default",
         include=None,
         exclude=None,
+        datetime_is_numeric=False,
     ):
+        if PANDAS_GT_110:
+            datetime_is_numeric_kwarg = {"datetime_is_numeric": datetime_is_numeric}
+        elif datetime_is_numeric:
+            raise NotImplementedError(
+                "datetime_is_numeric=True is only supported for pandas >= 1.1.0"
+            )
+        else:
+            datetime_is_numeric_kwarg = {}
 
         if self._meta.ndim == 1:
-            return self._describe_1d(self, split_every, percentiles, percentiles_method)
+
+            meta = self._meta_nonempty.describe(**datetime_is_numeric_kwarg)
```

Comment:
Should we pass other keyword arguments here like `include`/`exclude`? I suspect those might influence the resulting `meta` which is returned

Reply:
yeah good point. Let's do that.